scripts/index-sync-rotate.sh

```diff
@@ -35,6 +34,26 @@ LOG_PREFIX="[ $$ - $(date +"%F %T")] "
     # .spp
     # .sps
 
+
+
+json_logger() {
+    log_level=$1
+    timestamp=$(date --utc +%FT%T.%3NZ)
+    self=$(readlink -f "${BASH_SOURCE[0]}")
+    self=$(basename "$self")
+    jq --raw-input --compact-output --monochrome-output \
+    '{ "app":
+        {
+            "time": "'"${timestamp}"'",
+            "level": "'"${log_level}"'",
+            "logger": "'"${self}"'",
+            "pidTid": "'$$'",
+            "function": "'"${FUNCNAME[0]}"'",
+            "message": .
```

Comment:
Should not the `.` be between double quotes ?

Reply:
the `raw-input`which is read from stdin will be quoted automatically by jq:
```bash
echo "info" | json_logger INFO
{
  "app": {
    "time": "2022-09-22T06:59:26.146Z",
    "level": "INFO",
    "logger": "index-sync-rotate.sh",
    "pidTid": "15688",
    "function": "json_logger",
    "message": "info"
  }
}

echo "error" | json_logger ERROR
{
  "app": {
    "time": "2022-09-22T06:59:26.152Z",
    "level": "error",
    "logger": "index-sync-rotate.sh",
    "pidTid": "15688",
    "function": "json_logger",
    "message": "error"
  }
}
```

Reply:
👍🏼